AirCasting/SearchAndFollow/CompleteScreen/SearchSession.swift

```diff
@@ -0,0 +1,50 @@
+// Created by Lunar on 22/02/2022.
+//
+
+import Foundation
+
+struct SearchSession {
+    let name: String
+    let startTime: Date
+    let endTime: Date
+    let type: SessionType
+    let longitude: Double
+    let latitude: Double
+    let streams: [SearchSessionStream]
+    
+    struct SearchSessionStream: Identifiable {
+        var id: Int
+        let sensorPackageName: String?
+        let sensorName: String
+        let measurements: [SearchSessionMeasurement]
+        
```

Comment:
Can be deleted I think

Reply:
This will probably be usefull

Reply:
Oh, I was talking about the empty line, sorry